### PR TITLE
FP-823: Header Changes

### PR DESCRIPTION
**Pull request by gstvdesign**
# Ticket(s)

- [FP-823: Header Changes](https://fourkitchens.clickup.com/t/36718269/FP-823)

## Purpose

**This PR does the following:**

- Style updates to [address feedback](https://miro.com/app/board/uXjVKZJONU0=/)
### Functional Testing

- [ ] Visit the main header and search in storybook

### CODE REVIEW

**Comment by fkbender** [comment]
Task linked: [FP-823 Header Changes](https://app.clickup.com/t/36718269/FP-823)

File: src/lib/assets/css/main-menu.css [hunk 1]
```diff
@@ -115,4 +115,9 @@
   @apply max-lg:pt-0;
 }
 
+/* .menu-item--wrapper-button:last-child {
+  @apply mb-20 lg:mb-0;
```
Comment (mcortes19) [inline src/lib/assets/css/main-menu.css:119]: @gstvdesign if you are sure we don't need this, please remove

File: src/lib/components/01-elements/link/link.tsx [hunk 2]
```diff
@@ -21,7 +21,7 @@ export type LinkProps = AnchorHTMLAttributes<HTMLAnchorElement> & {
 };
 
 const colorLinkSchema = {
-  black: 'text-black hover:text-teal',
+  black: 'text-black hover:text-blue',
```
Comment (mcortes19) [inline src/lib/components/01-elements/link/link.tsx:24]: @gstvdesign just to double check if this change is correct?

**Review by mcortes19 — CHANGES_REQUESTED** [review]
@gstvdesign in general all of this look good, could you please address my comments before merge this

File: src/lib/components/01-elements/link/link.tsx (hunk 2 above)
Comment (gstvdesign) [reply src/lib/components/01-elements/link/link.tsx:24]: Confirmed with Victor. Black links should always be blue when hovered.

**Review by mcortes19 — APPROVED** [review]

**Pull request merged**